awpy/parser/parse_demo.go

```diff
@@ -136,6 +136,9 @@ type GameRound struct {
 	WinningTeam          *string            `json:"winningTeam"`
 	LosingTeam           *string            `json:"losingTeam"`
 	Reason               string             `json:"roundEndReason"`
+	MVPName              string             `json:"MVPName"`
+	MVPSteamID           int64              `json:"MVPSteamID"`
+	MVPReason            string             `json:"MVPReason"`
```

Comment:
I think according to the demoinfocs-golang documentation the player can be nil and so we should handle this like we do the other cases where that can happen.

https://pkg.go.dev/github.com/markus-wa/demoinfocs-golang/v3@v3.3.0/pkg/demoinfocs/events#RoundMVPAnnouncement

https://github.com/pnxenopoulos/awpy/blob/main/awpy/parser/parse_demo.go#L205

Reply:
Indeed, the comment by @JanEricNitschke is correct. We can achieve this simply by using the `*` to denote a pointer.